app/src/main/java/com/fish/monsters/features/history/ui/components/PointStartEnd.kt

```diff
@@ -0,0 +1,41 @@
+package com.fish.monsters.features.history.ui.components
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.DpSize
+import androidx.compose.ui.unit.dp
+import com.fish.monsters.common.shapes.PartiallyCutCornerShape
+import com.fish.monsters.core.theme.DarkPrimaryColor
+import com.fish.monsters.core.theme.DarkPrimaryColorA12
+
+@Composable
+fun PointStartEnd(
```

Comment:
To nie jest chyba `PointStartEnd` bo odpowiada ża wszystkie punkty we wzmocnieniach, moze bardziej `EnhancementPoint`?